engine/common/sys_con.c

```diff
@@ -123,8 +123,14 @@ static void Sys_FlushLogfile( void )
 void Sys_InitLog( void )
 {
 	const char	*mode;
+#if XASH_PSVITA
+	// force-enable logging when developer is set to 2+
+	const qboolean force_log = ( host_developer.value > DEV_NORMAL );
```

Comment:
Why should we do this? If you somehow enabled developer mode (using `-dev` flag I suppose, not switching cvar), I think you can enable `-log` as well.